bootstrap.sh

```diff
@@ -138,7 +143,7 @@ do
 	    # Bootstrap the autotool subsystems
 	    bootstrap aclocal$amver $acincludeflag
 	    bootstrap autoheader$acver
-	    bootstrap_libtoolize ${LIBTOOL_BIN}ize${ltver}
+	    bootstrap_libtoolize ${LIBTOOL_BIN}${ltver}
```

Comment:
Would not this break bootstrap where it was working before? For example, this used to get `glibtoolize` on Darwin but now will only get `glibtool`. Why would this continue to work on Darwin?

Reply:
Semaphore CI build [failure](https://semaphoreci.com/squid/squid/branches/pull-request-381/builds/1) increase my concerns about the `ize` suffix removal.